src/webpage.cpp

```diff
@@ -1057,18 +1058,23 @@ QString WebPage::renderBase64(const QByteArray& format)
     return "";
 }
 
-QImage WebPage::renderImage()
+QImage WebPage::renderImage(bool onlyViewport)
```

Comment:
Thanks @astefanutti! This solves my problem especially with 'responsive' pages. Earlier I had trouble rendering screenshots from sites like - http://html5up.net.

But I have to agree with @ariya, the bool can mislead. Also, the name onlyViewport is not self explanatory, it doesn't tell me what this parameter is going to do.